HandledError.py

```diff
@@ -0,0 +1,15 @@
+from flask import jsonify
+
+
+class HandledError(Exception):
+    def __init__(self, description: str, status_code: int, name: str):
+        self.description = description
+        self.status_code = status_code
+        self.name = name
+
+    def __str__(self):
+        return self.description
+
```

Comment:
What IDE do you use? VS code or PyCharm?

Reply:
PyCharm

Reply:
you have many blank lines in this files, while PEP has a standard on the number of blank lines (2 between function, 1 between methods, 1 at the end of file). PyCharm underlines such code-style problems with yellow. Please follow PyCharm code-style suggestions.

Reply:
fixed